app/data/Owners.scala

```diff
@@ -29,16 +24,22 @@ object Owners extends Owners {
 
   override def default = Owner("phil.wills")
 
-  override def stacks: Set[(String, SSA)] = Set(
-    "dotcom.platform" -> SSA("frontend"),
-    "simon.hildrew" -> SSA("deploy"),
-    "adam.fisher" -> SSA("security"),
-    "digitalcms.dev" -> SSA("flexible"),
-    "digitalcms.dev" -> SSA("workflow"),
-    "digitalcms.dev" -> SSA("cms-fronts"),
-    "digitalcms.dev" -> SSA("elk-new"),
-    "thegrid.dev" -> SSA("media-service")
-    //TODO: complete list
+  override def all: Set[Owner] = Set(
+    Owner("journalism.dev", ssas = Set(SSA("content-api"), SSA("content-api-preview")), accounts = Set("capi")),
+    Owner("dig.dev.tooling", ssas = Set(SSA("deploy")), accounts = Set("deploy-tools")),
+    Owner("digitalcms.dev", ssas = Set(
+      SSA("flexible"),
+      SSA("workflow"),
+      SSA("cms-fronts"),
+      SSA("elk-new")
+    ), accounts = Set("cmsFronts", "composer", "workflow", "media-service", "cmsSupport")),
```

Comment:
Whats the difference between `ssas` and `accounts` here? Perhaps there is a way to merge the groups together, so `flexible` with `composer`?
I think the `media-service` is missing an `ssa` and `elk-new` is missing an `account`?